### PR TITLE
Compliance Phase Working Branch

### PR DESCRIPTION
**Pull request by kagarmoe**
### Description

Sub-branch of https://github.com/chef/chef-web-docs/pull/2781

### Description

Sub-branch of https://github.com/chef/chef-web-docs/pull/2781

Preview:
https://deploy-preview-2870--chef-web-docs.netlify.app/chef_client_overview/
https://deploy-preview-2870--chef-web-docs.netlify.app/chef_compliance_phase/
https://deploy-preview-2870--chef-web-docs.netlify.app/glossary/

### Definition of Done

* Well organized
* Useful
* Instructional

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

File: content/chef_client_overview.md [hunk 1]
```diff
@@ -17,11 +17,14 @@ aliases = ["/chef_client_overview.html", "/chef_client.html"]
...
 
 For the `chef-client` command line tool, see
-[chef-client(executable)](/ctl_chef_client/) The Chef Infra Client
-executable can be run as a daemon.
+[chef-client(executable)](/ctl_chef_client/).
 
 {{< /note >}}
 
+**New in Chef Infra Client 17**
+Chef Infra Client's Compliance Phase lets you automatically execute compliance audits and view the results as part of any Chef Infra Client Run. The Compliance Phase of the Chef Infra Client run replaces the legacy audit cookbook and works with your existing audit cookbook attributes, and you can also set it up for new cookbooks. See [Compliance Phase]({{< relref "chef_compliance_phase" >}}) documentation to upgrade.
```
Comment (dishanktiwari2501) [inline content/chef_client_overview.md]: The second line "The Compliance Phase....." can be divided into two sentences. The "Chef Infra Client run replaces...." sounds a bit confusing and the "and you can also set......" is sounding like an extra add-on in a sentence.

File: content/glossary.md [hunk 2]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-    whether the submitted change should ship all the way out to its
-    final destination.
+: Agents are programs that you configure to perform actions continuously and autonomously.
 
-**analytics rules**
+Acceptance
 
-:   Event tracking during a Chef Infra Client run that generates data
-    made visible to Workflow.
+: A Workflow stage. The Acceptance stage is where your team decides whether the submitted change should ship all the way out to its final destination.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: This sentence sounds a bit interesting, like a script of a fictional story. Do we use these in our documentation?

File: content/glossary.md [hunk 3]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
 
-**Automate**
+analytics rules
 
-:   A full suite of enterprise capabilities for maintaining continuous
-    visibility into application, infrastructure, and security
-    automation.
+: Event tracking during a Chef Infra Client run that generates data made visible to Workflow.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: This can be rephrased.

File: content/glossary.md [hunk 4]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-    This product has been superseded by Chef Workstation which should be
-    used instead.
+: `chef` is the Chef Workstation command line tool for managing your Chef development environment including repositories, cookbooks, recipes, attributes, templates, custom resources, and Ruby dependencies.
 
-**Chef Infra Client**
+chef-client
 
-:   A command-line tool that that runs Chef. Also, the name of Chef as
-    it is installed on a node.
+: The `chef-client` is an agent that runs on nodes and a command line tool that runs on your local computer. The `chef-client` allows you to ensure the configuration of your systems through policy code. You install the chef-client on to nodes with the bootstrap process from your computer and configure it to run on an interval to ensure configuration compliance. You use the chef-client command line tool from your workstation to send instructions to the agent.
```
Comment (dishanktiwari2501) [inline content/glossary.md:34]: The third sentence, "You install the chef-client......" can be rephrased as the start sounds abrupt.

File: content/glossary.md [hunk 5]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
 
-**Chef Workstation**
+chef-apply
 
-:   A collection of tools to aide in development of Chef Infra
-    cookbooks. It uses the full stack installer to give you everything
-    you need to get going in one package. You can download it at [Chef
-    Workstation](https://downloads.chef.io/chef-workstation/).
+: A command-line tool that allows a single recipe to be run from the command line.
```
Comment (dishanktiwari2501) [inline content/glossary.md:38]: "to be run..." can be rephrased

File: content/glossary.md [hunk 6]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
 
-**chef-repo**
+ChefDK
 
-:   The repository structure in which cookbooks are authored, tested,
-    and maintained. View [an example of
-    the](https://github.com/chef/chef-repo) chef-repo.
+: The legacy package of tools for developing Chef Infra cookbooks. This product has been superseded by Chef Workstation which should be used instead.
```
Comment (dishanktiwari2501) [inline content/glossary.md:42]: The second sentence "This product has been....." can be rephrased as we are using "should" and "instead" in the same context.

File: content/glossary.md [hunk 7]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-**Chef Infra Server**
+Chef Infra Client
 
-:   The Chef Infra Server acts as a hub for configuration data. The Chef
-    Infra Server stores cookbooks, the policies that are applied to
-    nodes, and metadata that describes each registered node that is
-    being managed by Chef Infra Client. Nodes use Chef Infra Client to
-    ask the Chef Infra Server for configuration details, such as
-    recipes, templates, and file distributions.
+: A command-line tool that that runs Chef. Also, the name of Chef as it is installed on a node.
```
Comment (dishanktiwari2501) [inline content/glossary.md:46]: Double "that" in the sentence

File: content/glossary.md [hunk 8]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
+: A definition is code that is reused across recipes, similar to a compile-time macro, and is defined in a cookbook. Definitions are not recommended for new users and Custom Resources should be written for reusable code instead.
 
-**knife**
+Delivered
 
-:   A command-line tool that provides an interface between a local
-    chef-repo and the Chef Infra Server. Use it to manage nodes,
-    cookbooks, recipes, roles, data bags, environments, bootstrapping
-    nodes, searching the Chef Infra Server, and more.
+: A Workflow stage. Delivered is the final stage of the pipeline, what it means for your system is up to you. It could mean deploying the change so that it is live and receiving production traffic, or it might mean publishing a set of artifacts so they are accessible for your customers.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: "artifacts so they are accessible....." sounds abrupt.

File: content/glossary.md [hunk 9]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-    Infra.
+: A library allows arbitrary Ruby code to be included in a cookbook, either as a way of extending the classes that are built-in to Chef Infra Client or by implementing entirely new functionality.
 
-**ohai**
+Management Console
 
-:   Ohai is a tool that is used to detect attributes on a node, and then
-    provide these attributes to Chef Infra Client at the start of every
-    run.
+: The Chef web-based management console you can use to manage Role Based Access Control (RBAC), edit and delete nodes, and reset private keys. Keep up to date with what's happening during Chef Infra Client runs across an entire organization or on specific
```
Comment (dishanktiwari2501) [inline content/glossary.md]: The start can be rephrased as "console you can use......" sounds abrupt

File: content/glossary.md [hunk 10]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
+: A node object is a history of the attributes, run-lists, and roles that were used to configure a node that is under management by Chef Infra.
 
-**policy**
+ohai
 
-:   Policy settings can be used to map business and operational
-    requirements, such as process and workflow, to settings and objects
-    stored on the Chef Infra Server. See roles, environments, and data
-    bags.
+: Ohai is a tool that is used to detect attributes on a node, and then provide these attributes to Chef Infra Client at the start of every run.
```
Comment (dishanktiwari2501) [inline content/glossary.md:114]: "then" can be removed

File: content/glossary.md [hunk 11]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
+: A library allows arbitrary Ruby code to be included in a cookbook, either as a way of extending the classes that are built-in to Chef Infra Client or by implementing entirely new functionality.
 
-**ohai**
+Management Console
 
-:   Ohai is a tool that is used to detect attributes on a node, and then
-    provide these attributes to Chef Infra Client at the start of every
-    run.
+: The Chef web-based management console you can use to manage Role Based Access Control (RBAC), edit and delete nodes, and reset private keys. Keep up to date with what's happening during Chef Infra Client runs across an entire organization or on specific
+ nodes.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: The line has changed I believe.

File: content/glossary.md [hunk 12]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-    steps needed to bring that item to the desired state.
+: A recipe is a collection of resources that tells Chef Infra Client how to configure a node.
 
-**role**
+Rehearsal
 
-:   A role is a way to define certain patterns and processes that exist
-    across nodes in an organization as belonging to a single job
-    function.
+: If all phases of Union succeed, then the Rehearsal stage is triggered. Rehearsal increases confidence in the artifacts and the deployment by repeating the process that occurred in Union in a different environment.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: "then" can be removed

File: content/glossary.md [hunk 13]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
 
-**run-list**
+Reporting
 
-:   A run-list defines all of the configuration settings that are
-    necessary for a node that is under management by Chef to be put into
-    the desired state and the order in which these configuration
-    settings are applied.
+: Capture and visualize what happens during the execution of Chef Infra Client runs across all of your Chef-managed infrastructure.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: "of" can be removed

File: content/glossary.md [hunk 14]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
-    whole.
+: A role is a way to define certain patterns and processes that exist across nodes in an organization as belonging to a single job function.
 
-**Verify**
+run-list
 
-:   A Workflow stage. The purpose of Verify is to run checks so that the
-    system can decide if it's worth the time of a human to review the
-    submitted change.
+: A run-list defines all of the configuration settings that are necessary for a node that is under management by Chef to be put into the desired state and the order in which these configuration settings are applied.
```
Comment (dishanktiwari2501) [inline content/glossary.md:138]: Can we divide it into two sentences?

**Review by tas50 — APPROVED** [review]

File: content/glossary.md [hunk 15]
```diff
@@ -8,261 +8,185 @@ aliases = ["/glossary.html"]
...
+
+: A Workflow stage. The purpose of Verify is to run checks so that the system can decide if it's worth the time of a human to review the submitted change.
+
+visibility
+
+: A feature of Workflow that provides real-time visibility into what is happening on the Chef Infra Server, including what's changing, who made those changes, and when they occurred.
+
+workflow
+
+: A feature of Workflow that manages changes to both infrastructure and application code, giving your operations and development teams a common platform for developing, building, testing, and deploying cookbooks, applications, and more.
```
Comment (dishanktiwari2501) [inline content/glossary.md]: "manages" should be "manage"
"operations" should be "operation"

File: themes/docs-new/layouts/shortcodes/chef_client_run.md [hunk 16]
```diff
@@ -16,46 +19,55 @@ During every Chef Infra Client run, the following happens:
...
-<tr class="odd">
+<tr>
 <td><p><strong>Update the node object, process exception and report handlers</strong></p></td>
 <td><p>When all of the actions identified by resources in the resource collection have been done and Chef Infra Client finishes successfully, then Chef Infra Client updates the node object on the Chef Infra Server with the node object built during a Chef Infra Client run. (This node object will be pulled down by Chef Infra Client during the next Chef Infra Client run.) This makes the node object (and the data in the node object) available for search.</p>
 <p>Chef Infra Client always checks the resource collection for the presence of exception and report handlers. If any are present, each one is processed appropriately.</p></td>
 </tr>
-<tr class="even">
+<tr>
+<td><strong>Get, run Chef InSpec Compliance Profiles</strong></td>
+<td>After the Chef Infra Client run finishes, it begins the Compliance Phase, which is a Chef InSpec run  within the Chef Infra Client. Chef InSpec retrieves tests from either a legacy audit cookbook or a current InSpec profile.</td>
```
Comment (dishanktiwari2501) [inline themes/docs-new/layouts/shortcodes/chef_client_run.md]: an extra space between "run  within"

File: content/glossary.md (hunk 2 above)
Comment (kagarmoe) [reply content/glossary.md]: I'm pretty sure it will vanish in the next rebase. But it is out of date.

File: content/glossary.md (hunk 5 above)
Comment (kagarmoe) [reply content/glossary.md:38]: That hurts my soul to see!

File: content/glossary.md (hunk 14 above)
Comment (kagarmoe) [reply content/glossary.md:138]: Let's come back to this. It needs more careful thought.

**Review by dishanktiwari2501 — APPROVED** [review]

**Pull request merged**